download/releaseNotes/releaseNotes7.adoc

```diff
@@ -5,6 +5,15 @@
 :awestruct-release_notes_version: 7
 :awestruct-release_notes_version_qualifier: Final
 
+== New and noteworthy: Engine 7.38.0.Final
+
+=== Several major API deprecations
+
+With this release, we are announcing several major API deprecations.
+All users of OptaPlanner are discouraged from using these APIs in their projects, and we encourage everyone to check out our link:../upgradeRecipe/upgradeRecipe7.html[updated upgrade recipe] and adapt.
```

Comment:
Scientifically this is correct and thruth. Can we add a bit of psychology softness to it too?
The "major" is scary. Most users won't be affected by these deprecations, we should state that. In fact if 90% of the users are untouched by these deprecations, I don't think we can call them "major API deprecations"?
Suggestion:

```
=== Deprecation effort

With this release, we are deprecating some methods for which there are now better alternatives.
We encourage everyone to check out our link:../upgradeRecipe/upgradeRecipe7.html[updated upgrade recipe] to adapt accordingly.

That said, ...
```